unstract/flags/src/unstract/flags/feature_flag.py

```diff
@@ -31,7 +31,9 @@ def check_feature_flag_status(
     """
 
     try:
-        FLIPT_SERVICE_AVAILABLE = os.environ.get("FLIPT_SERVICE_AVAILABLE", False)
+        FLIPT_SERVICE_AVAILABLE = (
```

Comment:
@Deepak-Kesavan I believe this is used in one more place https://github.com/search?q=repo%3AZipstack%2Funstract%20FLIPT_SERVICE_AVAILABLE&type=code

can you change it there as well?